src/components/Button/Button.stories.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';

```

Comment:
How do you feel about using an [action decorator](https://www.npmjs.com/package/@storybook/addon-actions#action-decorators) to create an `eventAction` that only logs `args[0].target` (i.e. the clicked button)?

Reply:
It seems to be [a Firefox bug](https://github.com/jquery/PEP/issues/111). What version of FF are you running?

Reply:
OK, thanks for the hint. I'm not sure which version I have (can't check it right now) - probably not the latest. 
But I'll check if it works with Chrome and if it's browser related I think we need to add a weak-map polyfill or check if we can configure Babel to handle it.

Reply:
The problem with using a polyfill is that Firefox technically supports `WeakMap`, it’s jut that the implement in those versions is broken.